src/ConnectedMode/Embedded/SonarLintTargets.xml

```diff
@@ -1,4 +1,59 @@
 ﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup>
+<!-- TODO 
+
+* decide the final root directory location
+* remove the debug Target
+* decide whether to include $(DesignTimeBuild) in the Conditions.
+
+-->
+
+  <!-- 
```

Comment:
The `TODO` comment above should be removed.

The comment below would remain so anyone looking at the file knows what it is for.
We might want to include a link to the wiki/docs.